app/controllers/runtime/organizations_controller.rb

```diff
@@ -285,9 +285,26 @@ def user_guid_parameter
     end
 
     def after_create(organization)
-      return if SecurityContext.admin?
-      organization.add_user(user)
-      organization.add_manager(user)
+      unless SecurityContext.admin?
+        organization.add_user(user)
+        organization.add_manager(user)
+      end
+
+      organization.users.each do |user|
+        @user_event_repository.record_organization_role_add(organization, user, 'user', UserAuditInfo.from_context(SecurityContext), request_attrs)
+      end
+
+      organization.auditors.each do |auditor|
```

Comment:
Since our code only looks to add them as a "user" and "manager", do we need to: a) Loop through all possible users of type {user/billing manager/audit/manager) b) try to record events for the other types not explicitly added above (billing manager & auditor)